### PR TITLE
Remove okcomments from InputIllegalTypeGregCal

### PR DESCRIPTION
**Pull request by RatikaDogra**
Fixed this issue
Issue: #13213

### CODE REVIEW

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCodeEqualsParameter.java [hunk 1]
```diff
@@ -7,7 +7,7 @@
 package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
 
 public class InputEqualsHashCodeEqualsParameter {
-    public static class TestClass1 { // ok
+    public static class TestClass1 {
```
Comment (MANISH-K-07) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCodeEqualsParameter.java:10]: The files lgtm....

Squash and rebase to single commit.

Please go through the Issue https://github.com/checkstyle/checkstyle/issues/13213 description before sending PR.
You must also remove the respective suppression from [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml)

**Review by MANISH-K-07 — COMMENTED** [review]
Your commit message needs to adhere to checkstyle rules : 

```
[ERROR] Failures: 
[ERROR]   CommitValidationTest.testCommitMessageHasProperStructure:216 Commit a4bb9d93228dc9f7ebc25fb8d545e7a301bdb014 message: "fix: remove ok comments from InputEqualsHashCode" is invalid
Proper commit message should adhere to the following rules:
    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters

The rule broken was: 1

```

Please change accordingly.
Look at merged PRs for example : 

- #14259 
- #14260 
- #14264

**Comment by rdiachenko** [comment]
> This PR can be closed as it is solved by this PR - #14452

You are linking to the current PR.

@RatikaDogra please take a look at the commets above and try to fix this PR.

**Comment by RatikaDogra** [comment]
Okay

**Comment by nrmancuso** [comment]
We have lost contact with author, anyone is welcome to reuse changes here to close the issue.

**Pull request closed without merging**